modules/appeals_api/spec/models/higher_level_review_spec.rb

```diff
@@ -341,8 +341,102 @@
     describe '#claimant' do
       subject { higher_level_review_v2.claimant }
 
-      it do
-        expect(subject.class).to eq AppealsApi::NonVeteranClaimant
+      it { expect(subject.class).to eq AppealsApi::Appellant }
+    end
+
+    describe '#veteran' do
+      subject { higher_level_review_v2.veteran }
+
+      it { expect(subject.class).to eq AppealsApi::Appellant }
+    end
+
+    describe '#signing_appellant' do
```

Comment:
This makes the assumption that we want to sign with the Claimant's name if a claimant is present. If not, let's discuss and update.